realsync

```diff
@@ -124,6 +124,10 @@ sub mainloop {
         $FILE_IDENTITY = $CONFIG{identity};
     }
 
+    # Set the number of changes that will trigger an rsync from the config file
+    $MIN_CHANGES_FOR_RSYNC = cfg("rsync_threshold");
```

Comment:
this should have a default value, or realsync will break for people who don't have this option in their config yet?